test/SILGen/default_arguments.swift

```diff
@@ -148,6 +150,9 @@ testMagicLiterals()
 closure { testMagicLiterals() }
 autoclosure(testMagicLiterals())
 
+// CHECK: string_literal utf16 "default_arguments"
```

Comment:
Thanks! Is there no function for this one?

Reply:
It ends up in 
```
sil @main : $@convention(c) (Int32, UnsafeMutablePointer<Optional<UnsafeMutablePointer<Int8>>>)
```

and actually I just realized it will have a duplicate string literal
```
  %9 = string_literal utf16 "default_arguments"   // user: %14
```
and the one we care about here
```
%36 = string_literal utf16 "default_arguments"  // user: %41
```

Reply:
Ah, right, in a script file with top-level code this is treated like a local variable, not as a lazy global. Oh well, just checking that it doesn't crash is probably good enough. I won't ask you to add a new test file.